strongbox-web-forms/src/main/java/org/carlspring/strongbox/forms/cron/CronTaskConfigurationForm.java

```diff
@@ -0,0 +1,100 @@
+package org.carlspring.strongbox.forms.cron;
+
+import org.carlspring.strongbox.validation.cron.UniqueCronTaskConfiguration;
+
+import javax.validation.constraints.NotEmpty;
+import java.io.Serializable;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.common.collect.Maps;
+
+/**
+ * @author Pablo Tirado
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CronTaskConfigurationForm
+        implements Serializable
+{
+
+    @NotEmpty(message = "Configuration UUID is required!")
+    @UniqueCronTaskConfiguration(groups = NewConfiguration.class, message = "Configuration UUID is already taken.")
+    private String uuid;
```

Comment:
This form field is unnecessary, because the `CronTaskDataServiceImpl#save` method already generates a `UUID` when we're creating a new `CronTask`. When updating/deleting a cron task you will receive the `UUID` from the `URL` and when you are creating a new one you will not actually be sending a `UUID` as it will be generated automatically.

As for the uniqueness check, we should probably move it into `CronTaskDataServiceImpl#save` and return a `CronTaskUUIDNotUniqueException`, perhaps? (@carlspring thoughts?)

Reply:
I agree. I think that would be a better place. The service should be handling this logic.